README.md

```diff
@@ -7,3 +7,21 @@ This is a collection of Web Audio API examples.
  - [Stress Test](http://googlechromelabs.github.io/web-audio-samples/stress-test/boxes)
 
 __NOTE__: Some of these samples are not functional or cross-browser compatible. Please feel free to file issues or submit PRs.
+
+
+## Support
+
+If you have found an error in this library, please file an issue at: https://github.com/GoogleChromeLabs/web-audio-samples/issues.
+
+Patches are encouraged, and may be submitted by forking this project and submitting a pull request through GitHub. See CONTRIBUTING for more detail.
```

Comment:
Wrap lines?  Does "CONTRIBUTING" link to the doc?

Reply:
This is a template text from other Google open-source project. (e.g. Omnitone) I would like to keep it as is.